apps/portals/e2e/configs/routesConfig.ts

```diff
@@ -17,13 +17,7 @@ type RouteConfig = Record<Portal, RouteButtonsLinks>
 const routesConfig: RouteConfig = {
   adknowledgeportal: {
     buttons: ['Home', 'Explore', 'Sign In'],
-    links: [
-      'Analytical Workspace',
-      'Data Access',
-      'Contribute',
-      'News',
-      'Help',
-    ],
+    links: ['Analysis Platforms', 'Data Access', 'Contribute', 'News', 'Help'],
```

Comment:
I broke these tests in a previous PR

Reply:
Hallie fixed in https://github.com/Sage-Bionetworks/synapse-web-monorepo/pull/634